### PR TITLE
edition layout: fetch multiple works edition

### PR DESCRIPTION
**Pull request by jum-s**
address #361 

Since multiple works edition is a small fraction of the total amount of editions, and since a work with subentities might be a big chunk of data, the advantages of mutualizing works requests looks less important than the simplicity of having one work subentities request per component.

### CODE REVIEW

File: app/modules/entities/components/layouts/other_editions.svelte [hunk 1]
```diff
@@ -52,21 +46,22 @@
...
       />
-	  </div>
+    </div>
 	{/if}
 {/await}
 
 <style lang="scss">
   @import '#general/scss/utils';
-  .other-editions{
+  .other-edition{
```
Comment (maxlath) [inline app/modules/entities/components/layouts/other_editions.svelte:59]: The plural was fine no? It's still other editions from one of the edition works
```suggestion
  .other-editions{
```

File: app/modules/entities/components/layouts/other_editions.svelte [hunk 2]
```diff
@@ -31,12 +30,7 @@
   </div>
 {:then}
 	{#if isNonEmptyArray(otherEditions)}
-	  <div class="other-editions">
-	    <div class="editions-list-title">
-	      <h5>
-	        {I18n('other editions')}
-	      </h5>
-	    </div>
+    <div class="other-edition">
```
Comment (maxlath) [inline app/modules/entities/components/layouts/other_editions.svelte:33]: Those dives just stack vertically, not using the available horizontal space on large screens. Could those `div.other-editions` be regrouped in a flex-row-wrapped list?
```suggestion
   <ul class="works-other-editions">
      <li class="other-editions">
```
```css
.works-other-editions{
  @include display-flex(row, baseline, center, wrap);
}
```

**Review by maxlath — CHANGES_REQUESTED** [review]

File: app/modules/entities/components/layouts/other_editions.svelte (hunk 2 above)
Comment (jum-s) [reply app/modules/entities/components/layouts/other_editions.svelte]: done in 4f9a137

File: app/modules/entities/components/layouts/other_editions.svelte (hunk 1 above)
Comment (jum-s) [reply app/modules/entities/components/layouts/other_editions.svelte]: done in 083836a

**Review by maxlath — APPROVED** [review]

**Pull request merged**